### DISCUSSION TITLE
ch10/ch10-07

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#173] [General]
# ch10/ch10-07

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch10/ch10-07.html

**Comment by zhangbest5** [comment]
godoc -http :8000 这我理解，是启动了本地文档服务；
其中-analysis=type和-analysis=pointer命令行标志参数用于打开文档和代码中关于静态分析的结果。
这 -analysis=type和-analysis=pointer 是在哪里用？没理解，也没查到

**Comment by Lambert-Rao** [comment]
这难道不应该是第一章就应该说的包管理吗，为什么第十章才讲

**Reply by mefly2012** [reply]
的确应该后面讲工具和包，因为应该先了解基础语法语义，先玩玩，写大点的工程才需要用到包

**Comment by a-firefly** [comment]
**go 1.24.2的标准包有 344 个。**
```shell
~ go list std | wc -l
     344
```

**Comment by Zacharia2** [comment]
windows powershell写法
```powershell
(go list std).Count
```

**Comment by Zacharia2** [comment]
windows powershell写法
```powershell
go list -f '{{join .Deps "\n"}}' strconv
go list -f '{{.ImportPath}} -> {{join .Imports " "}}' compress/...
```
